gh_pages/_source/session6/Docker-AWS.md

```diff
@@ -65,7 +65,7 @@ In terminal 1 we will load a robot to the parameter server
 ```
 mkdir -p abb_ws/src
 git clone -b kinetic-devel https://github.com/ros-industrial/abb.git
-docker run -v "/home/aderic/roscloud/training/abb_ws:/abb_ws" --network=host -it rosindustrial/core:kinetic /bin/bash
+docker run -v "/home/aderic/roscloud/training/abb_ws:/abb_ws" --network=host -it rosindustrial/core:melodic /bin/bash
```

Comment:
It points to a personal directory, it seems we never really took notice of this.  I don't think there's a melodic rosindustrial docker image

Reply:
Will you be changing the local directory (`/home/aderic/roscloud/training/abb_ws`) so that it matches the vm?

Reply:
In this file you'd change all instances of `/home/aderic` with `/home/ros-industrial/` as this is the  melodic VM's home directory.